services/madoc-ts/src/frontend/admin/pages/crowdsourcing/projects/project.tsx

```diff
@@ -42,7 +42,7 @@ export const Project: UniversalComponent<ProjectType> = createUniversalComponent
             { label: t('Configuration'), link: `/projects/${data.id}/configuration` },
             { label: t('Model'), link: `/projects/${data.id}/model` },
             { label: t('Crowdsourcing'), link: `/projects/${data.id}/tasks` },
-            { label: t('Search'), link: `/projects/${data.id}/search` },
+            { label: t('search index'), link: `/projects/${data.id}/search` },
```

Comment:
The other changes here are mostly capitalizing the first letter - should that be done here too?

Reply:
Ah good spot!